shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2513,6 +2541,17 @@ - (void)takeKeyboardScreenshotAndDisplay {
   _keyboardViewContainer.frame = _keyboardRect;
 }
 
+- (void)showEditMenu:(NSDictionary*)dictionary {
+  if (@available(iOS 16.0, *)) {
+    NSDictionary<NSString*, NSNumber*>* encodedTargetRect = dictionary[@"target_rect"];
+    CGRect globalTargetRect = CGRectMake(
+        [encodedTargetRect[@"x"] doubleValue], [encodedTargetRect[@"y"] doubleValue],
+        [encodedTargetRect[@"width"] doubleValue], [encodedTargetRect[@"height"] doubleValue]);
+    CGRect localTargetRect = [self.hostView convertRect:globalTargetRect toView:_activeView];
+    [_activeView showEditMenuWithTargetRect:localTargetRect];
```

Comment:
This PR should be something ready for you to pull and try out. I tested with this: 

```
    _channel.invokeMethod<void>(
      'TextInput.showSystemContextMenu',
      <String, dynamic>{
        // global coords
        'target_rect': <String, dynamic>{
          'x': 0,
          'y': 100,
          'width': 100,
          'height': 50,
        }
      },
    );
```

Reply:
note that i also changed the channel name, since "edit" menu is an ios term.